netbox/project-static/src/savedFiltersSelect.ts

```diff
@@ -0,0 +1,27 @@
+import {isTruthy} from "./util";
```

Comment:
This and some other lines within the file trigger complaints from `prettier`.

```suggestion
import { isTruthy } from './util';
```